ledger/src/blockstore.rs

```diff
@@ -766,6 +935,114 @@ impl Blockstore {
         );
     }
 
+    /// Collects and reports [`BlockstoreRocksDbColumnFamilyMetrics`] for the
+    /// all the column families.
+    pub fn submit_rocksdb_cf_metrics_for_all_cfs(&self) {
+        self.submit_rocksdb_cf_metrics::<cf::SlotMeta>("blockstore_rocksdb_cfs,cf_name=slot_meta");
```

Comment:
We can reduce the repeated "blockstore_rocksdb_cfs" with a macro, something like:
```
macro_rules! rocksdb_metric_with_tag {
    ($tag_name: literal) => {
        concat!("blockstore_rocksdb_cfs,cf_name=", $tag_name)
    };
}
```

Using the macro, this would then become:
```
self.submit_rocksdb_cf_metrics::<cf::SlotMeta>(rocksdb_metric_with_tag!("slot_meta"));
```

Reply:
Neat!  Totally forgot I can use macros to work around the static string combination.  Will do!